app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -53,6 +53,20 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
     final List<GithubGoldStatusUpdate> statusUpdates = <GithubGoldStatusUpdate>[];
     log.debug('Beginning Gold checks...');
     await for (PullRequest pr in gitHubClient.pullRequests.list(slug)) {
+      // Check when this PR was last updated. Gold does not keep results after
+      // >20 days. If a PR has gone stale, we should draw attention to it to be
+      // updated or closed.
+      final DateTime updatedAt = pr.updatedAt;
```

Comment:
Do we need to take into account timezones and/or UTC in this logic?

Reply:
FYI: The Gold API returns timestamps in UTC (with a 'Z' suffix).